internal/updater/providers.go

```diff
@@ -22,12 +22,16 @@ func (u *Updater) updateProvider(ctx context.Context, provider Provider,
 	existingServersCount := u.storage.GetServersCount(providerName)
 	minServers := int(minRatio * float64(existingServersCount))
 	servers, err := provider.FetchServers(ctx, minServers)
+	providerWireguardKeyUnknown := false
+	if providerName == "private internet access" {
```

Comment:
use the constant `providers.PrivateInternetAccess` instead of writing a string directly